src/sssom/writers.py

```diff
@@ -99,20 +99,20 @@ def write_rdf(
     print(t.decode(), file=file)
 
 
-# todo: not sure the need for serialization param here; seems superfluous for some of these funcs
-def write_fhir_json(msdf: MappingSetDataFrame, output: TextIO, serialisation="fhir") -> None:
-    """Write a mapping set dataframe to the file as FHIR ConceptMap JSON."""
-    data = to_fhir_json(msdf)
-    json.dump(data, output, indent=2)
-
-
 def write_json(msdf: MappingSetDataFrame, output: TextIO, serialisation="json") -> None:
     """Write a mapping set dataframe to the file as JSON."""
-    if serialisation == "json":
-        data = to_json(msdf)
-        json.dump(data, output, indent=2)
-    else:
-        raise ValueError(f"Unknown json format: {serialisation}, currently only json supported")
+    func_map: Dict[str, Callable] = {
+        "fhir_json": to_fhir_json,
```

Comment:
Add comments what each of these serialisations is, ideally a link to some docs page

Reply:
I don't know of a docs page, but I'll take a look and see if there is anything. I assume you want me to link the docs at their live url, not use like relative pathing to the local markdown files. 
 
Ask for your comments about deprecating the old methods https://github.com/mapping-commons/sssom-py/pull/285#discussion_r1593689345 that is a very good idea. I didn't even think about that as I'm not used to maintaining the public libraries but I will do this thanks for suggesting!

Reply:
- [x] Done

I don't know anything about the `json` and `ontoportal_json` formats but I did my best.

<details><summary>Details</summary>
<p>

```py
def write_json(msdf: MappingSetDataFrame, output: TextIO, serialisation="json") -> None:
    """Write a mapping set dataframe to the file as JSON.

    :param serialisation: The JSON format to use. Supported formats are:
     - fhir_json: Outputs JSON in FHIR ConceptMap format (https://fhir-ru.github.io/conceptmap.html)
       https://mapping-commons.github.io/sssom-py/sssom.html#sssom.writers.to_fhir_json
     - json: Outputs to SSSOM JSON https://mapping-commons.github.io/sssom-py/sssom.html#sssom.writers.to_json
     - ontoportal_json: Outputs JSON in Ontoportal format (https://ontoportal.org/) 
       https://mapping-commons.github.io/sssom-py/sssom.html#sssom.writers.to_ontoportal_json
    """
```

</p>
</details>